markets/perps-market/contracts/storage/PerpsMarket.sol

```diff
@@ -47,6 +49,7 @@ library PerpsMarket {
         uint128 lastUtilizedLiquidationCapacity;
         // accountId => asyncOrder
         mapping(uint => AsyncOrder.Data) asyncOrders;
+        SetUtil.UintSet asyncOrdersSet;
```

Comment:
using a `UintSet` is very expensive as it creates an array and a mapping.  Generally speaking, we shouldn't be adding additional storage variables to contracts if its not serving the contract itself.  View functions are ok, but adding storage is not a negligible cost unfortunately.